### ISSUE TITLE
IMAP parameters missing?

### ISSUE THREAD

**Issue by jsbien** [vandry/mairix#37]
To connect to my IMAP I need in Gnus `smtpmail-stream-type  'starttls` and `smtpmail-smtp-service 587`. I am not aware of an equivalent for .mairixrc.

**Comment by vandry** [comment]
Both of the things you mention sound like they are related to SMTP, not IMAP. mairix does not interact with SMTP at all. Are you sure you need those parameters?

**Comment by jsbien** [comment]
You are right, the quoted parameters are not relevant. I just need to switch the port to 993. How?
Perhaps you will find this useful:
https://chatgpt.com/share/671b0642-a810-800d-84cf-548dac2739f3
